nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -72,12 +79,34 @@
  */
 public class TreeCommentManager extends AbstractCommentManager {
 
+    private static final Logger log = LogManager.getLogger(TreeCommentManager.class);
+
     protected static final String GET_COMMENT_PAGE_PROVIDER_NAME = "GET_COMMENT_AS_EXTERNAL_ENTITY";
 
     protected static final String GET_COMMENTS_FOR_DOCUMENT_PAGE_PROVIDER_NAME = "GET_COMMENTS_FOR_DOCUMENT_BY_ECM_PARENT";
 
     public static final String SERVICE_WITHOUT_IMPLEMENTATION_MESSAGE = "This service implementation does not implement deprecated API.";
 
+    /**
+     * The key to the config turning on or off autosubscription.
+     */
+    public static final String AUTOSUBSCRIBE_CONFIG_KEY = "org.nuxeo.ecm.platform.comment.service.notification.autosubscribe";
+
+    /**
+     * Counts how many comments where made on a specific document.
+     */
+    protected static final String QUERY_GET_COMMENTS_UUID_BY_COMMENT_ANCESTOR = //
+            "SELECT " + ECM_UUID + " FROM Comment WHERE " + ECM_ANCESTORID + " = '%s'";
+
+    /**
+     * Counts how many comments where made by a specific user on a specific document.
+     */
+    protected static final String QUERY_GET_COMMENTS_UUID_BY_COMMENT_ANCESTOR_AND_AUTHOR = //
+            QUERY_GET_COMMENTS_UUID_BY_COMMENT_ANCESTOR + " AND " + COMMENT_AUTHOR + " = '%s'";
+
+    /** @since 11.1 **/
+    public static final String COMMENT_RELATED_TEXT_ID = "commentRelatedTextId_%s";
```

Comment:
Cherry pick issue I think, the feature is only available on master.

Reply:
Yes this is related to full text search. I'll remove it.